src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
-export interface Figure {
+type FigureShape = 'triangle' | 'circle' | 'rectangle';
+
+type FigureColor = 'red' | 'green' | 'blue';
```

Comment:
as it is not stored in some constants file and you have this file only internally in the file - makes no sense to be that specific in the type naming
```suggestion
type Shape = 'triangle' | 'circle' | 'rectangle';

type Color = 'red' | 'green' | 'blue';
```